### ISSUE TITLE
Feature request: "keybase log view" to show logs

### ISSUE THREAD

**Issue by specious** [keybase/client#10031]
I know `keybase log send` is a well known way to have the logs sent to the Keybase team. However, it took me a minute to figure out where I can see the logs as a user.

The first place I checked was `keybase help log`, only to find out that `send` is the only available subcommand.

Then I checked `/var/log` and was disappointed to find no keybase logs there.

What happened then was @reub's [attached console output](https://github.com/keybase/client/issues/8894#issuecomment-335224638) in #8894 tipped me off that keybase has plist files in `~/Library/LaunchAgents` and those files happen to contain explicit references to their respective logs situated in `~/Library/Logs`.

So now I'm enjoying the magnificently detailed chronicles of the operation of the keybase components which currently have the privilege of taking part in the runtime of my system:

```
$ ls -l ~/Library/Logs/keybase*
-rw-------  1 specious  staff   51646090 Dec 22 17:38 /Users/specious/Library/Logs/keybase.kbfs.log
-rw-------  1 specious  staff    7578670 Dec 22 17:25 /Users/specious/Library/Logs/keybase.service.log
-rw-------  1 specious  staff  134217923 Dec 22 12:55 /Users/specious/Library/Logs/keybase.service.log-20171222T125345-0500-20171222T125547-0500
-rw-r--r--  1 specious  staff          0 Nov  3 17:29 /Users/specious/Library/Logs/keybase.start.log
-rw-r--r--  1 specious  staff     885211 Dec 22 16:53 /Users/specious/Library/Logs/keybase.updater.log
```

Still curious if the desktop GUI and the CLI utility keep their own logs in a different location, or if the service daemon is responsible for logging everything on their behalf...

Now, I'm not saying I'm the smartest keybase user or that I'm certain that information about where to look isn't posted somewhere obvious that I didn't check.  However, my understanding is that Keybase's mission is to assure users of the software's transparency, so I'd like to float the idea of including a subcommand in the `keybase log` function such as `show` or `view` that will allow users to easily retrieve the logs without having to figure out where they are stored on their system.

This will also provide a useful abstraction over the difference regarding the location of the logs on the various platforms such as macOS, Linux, Windows and (I'm hoping) mobile devices that have legitimately useful command line support in the future.

**Comment by specious** [comment]
The official document on **Understanding the Keybase filesystem** [mentions](https://keybase.io/docs/kbfs/understanding_kbfs#debugging) the locations of the kbfs and service log files on macOS and also recommends using Console.app to access the logs.

That's very helpful, especially to those who run across it.

**Comment by specious** [comment]
tl;dr - here's the best answer:

```
$ keybase status | ag log:
    log:       /Users/specious/Library/Logs/keybase.kbfs.log
    log:       /Users/specious/Library/Logs/keybase.service.log
    log:       /Users/specious/Library/Logs/keybase.updater.log
    log:       /Users/specious/Library/Logs/Keybase.app.log
```

**Comment by nealmcb** [comment]
Thanks!
Note that the "ag" command seems to be some sort of grep, which can be substituted.

**Comment by NewAlexandria** [comment]
This was useful, it helped us audit some interactions.  Thanks.
